main.py

```diff
@@ -81,7 +82,8 @@ def hello(i): return i == 'hi' or i == 'hey' or i == 'hello' or i == 'welcome'
     [text == "f", [custom_emoji["f_in_chat"], custom_emoji["f_keyboard"]]],
     ["love" in text or "heart" in text, [custom_emoji["cat_heartbongo"]]],
     ["🥳" in text or "party" in text or "woo" in text, [custom_emoji["party_blob"]]],
-    [True in [hello(i) for i in split], [emoji["wave"]]]
+    [True in [hello(i) for i in split], [emoji["wave"]]],
+    [zoheb, [custom_emoji["drinking milk"]]]
```

Comment:
updated it so it fit the new structure

custom_emoji maps to a emojis in the discord server, so make sure to choose a emoji in the discord right now, for example, if you want ur emoji to be :choccy_milk:, then you would add "choccy_milk" to the `custom_names` list and then change this link to say `[custom_emoji["choccy_milk"]]`